appium/webdriver/webdriver.py

```diff
@@ -176,6 +214,16 @@ def __init__(
         By.IMAGE = MobileBy.IMAGE
         By.CUSTOM = MobileBy.CUSTOM
 
+        for extension in extensions:
+            instance = extension(self.execute)
+            if hasattr(WebDriver, instance.method_name()):
+                logger.warn(
+                    '{} is alreadt defined. The new one is overriding the old one.'.format(instance.method_name())
+                )
+            setattr(WebDriver, instance.method_name(), instance.command_wrapper)
```

Comment:
should we actually invoke `command_wrapper` here? I'm not sure about what this method is supposed to return